identity-admin/webapp/components/search/Pagination.tsx

```diff
@@ -28,6 +29,17 @@ const Pagination = ({ currentPage, pageCount }: Props): JSX.Element => {
       ) : (
         <span className={firstLinkClassName + ' ' + disabled}>Previous</span>
       )}
+      {currentPage > 3 && (
```

Comment:
This is getting a little "magic numbers". Could we start using variables for these? Something like:

```ts
const MAX_PAGES = 4;

currentPage >= MAX_PAGES && // ...
```